download_dependencies.bat

```diff
@@ -1,36 +1,66 @@
+@echo off
+:: try to set compatibility with previous version by @bjornblissing
+IF "%dest%"=="" (SET dest=..\osg-3rdparty)
+IF EXIST "c:/3rdparty/" (set dest=c:/3rdparty)
+set sevenZip=7z
+IF EXIST "C:\Program Files\7-Zip\7z.exe" (set sevenZip=C:\Program Files\7-Zip\7z.exe)
```

Comment:
This will break on platforms where the x64 binary is installed. Try using `FIND_PROGRAM` instead:
https://cmake.org/cmake/help/latest/command/find_program.html